addons/io_scene_gltf2/blender/exp/gltf2_blender_extract.py

```diff
@@ -617,6 +617,11 @@ def extract_primitives(glTF, blender_mesh, blender_object, blender_vertex_groups
             triangles = tessellate_polygon((polyline,))
 
             for triangle in triangles:
+                if bpy.app.version < (2, 81, 15):
+                    # tessellate_polygon winding-order is reversed in old versions of Blender
+                    # See https://developer.blender.org/T70594
+                    triangle = (triangle[1], triangle[0], triangle[2])
```

Comment:
The [previous code here](https://github.com/KhronosGroup/glTF-Blender-IO/commit/2fb64552718638d8d4a1b80ebd6f82c9d4b293fc) was `0, 2, 1`, which I realize is the same order as `1, 0, 2`.  But could we use the original just for consistency?

Also I think you need an `else` clause around the code below.

Reply:
Yeah.